Sources/MapboxNavigation/Resources/pl.lproj/Localizable.strings

```diff
@@ -2,275 +2,274 @@
-"ROUTE_QUALITY_FEEDBACK" = "Route quality";
+"ROUTE_QUALITY_FEEDBACK" = "Jakość trasy";
 
 /* Specific route feedback that route suggested an illegal roadway. */
-"ROUTE_QUALITY_ILLEGAL_ROUTE_CARS_NOT_ALLOWED_FEEDBACK" = "Cars not allowed on street";
+"ROUTE_QUALITY_ILLEGAL_ROUTE_CARS_NOT_ALLOWED_FEEDBACK" = "Samochody nie są dozwolone na ulicy";
 
 /* General route feedback that route contained illegal instructions. */
-"ROUTE_QUALITY_ILLEGAL_ROUTE_FEEDBACK" = "Illegal route";
+"ROUTE_QUALITY_ILLEGAL_ROUTE_FEEDBACK" = "Nielegalna trasa";
 
 /* Specific route feedback that route travelled wrong way on a one-way road. */
-"ROUTE_QUALITY_ILLEGAL_ROUTE_ONE_WAY_FEEDBACK" = "Routed down a one way";
+"ROUTE_QUALITY_ILLEGAL_ROUTE_ONE_WAY_FEEDBACK" = "Trasa prowadzi przez drogę jednokierunkową";
 
 /* Specific route feedback that route suggested an illegal turn. */
-"ROUTE_QUALITY_ILLEGAL_ROUTE_TURN_NOT_ALLOWED_FEEDBACK" = "Turn wasn't allowed";
+"ROUTE_QUALITY_ILLEGAL_ROUTE_TURN_NOT_ALLOWED_FEEDBACK" = "Zakazany skręt";
 
 /* Specific route feedback that route contained non-existant roads. */
-"ROUTE_QUALITY_INCLUDED_MISSING_ROADS_FEEDBACK" = "Route includes missing roads";
+"ROUTE_QUALITY_INCLUDED_MISSING_ROADS_FEEDBACK" = "Trasa zawiera nieistniejące drogi";
 
 /* Specific route feedback that route was not drivable. */
-"ROUTE_QUALITY_NON_DRIVABLE_FEEDBACK" = "Route non drivable";
+"ROUTE_QUALITY_NON_DRIVABLE_FEEDBACK" = "Trasa niedostępna";
 
 /* Specific route feedback that route was not ideal according to user. */
-"ROUTE_QUALITY_NOT_PREFERRED_FEEDBACK" = "Route not preferred";
+"ROUTE_QUALITY_NOT_PREFERRED_FEEDBACK" = "Trasa niepreferowana";
 
 /* General route feedback that route contained closed road. */
-"ROUTE_QUALITY_ROAD_CLOSURE_FEEDBACK" = "Road closure";
+"ROUTE_QUALITY_ROAD_CLOSURE_FEEDBACK" = "Zamknięta droga";
 
 /* Specifc route feedback that route contained road that is permanently closed. */
-"ROUTE_QUALITY_ROAD_CLOSURE_PERMANENT_FEEDBACK" = "Street permanently blocked off";
+"ROUTE_QUALITY_ROAD_CLOSURE_PERMANENT_FEEDBACK" = "Ulica na stałe zablokowana";
 
 /* Specific route feedback that route contained impassible, narrow roads. */
-"ROUTE_QUALITY_ROADS_TOO_NARROW_FEEDBACK" = "Route had roads too narrow to pass";
+"ROUTE_QUALITY_ROADS_TOO_NARROW_FEEDBACK" = "Trasa zawiera zbyt wąskie drogi";
 
 /* Alternatives selection note about equal travel distance. */
-"SAME_DISTANCE" = "Same distance";
+"SAME_DISTANCE" = "Ta sama odległość";
 
 /* Alternatives selection note about equal travel time. */
-"SAME_TIME" = "Same time";
+"SAME_TIME" = "Ten sam czas";
 
 /* Alternatives selection note about a shorter route distance in any unit. */
-"SHORTER_ALTERNATIVE" = "%@ shorter";
+"SHORTER_ALTERNATIVE" = "%@ krótsza";
 
 /* Alternatives selection note about a slower route time interval in any unit. */
-"SLOWER_ALTERNATIVE" = "%@ slower";
+"SLOWER_ALTERNATIVE" = "%@ wolniejsza";
```

Comment:
need whole sentence, because all of those means the same
"%@ wolniejsza"" or "%@ wolniejszy"" or "%@ wolniej" etc